### PR TITLE
market, dock and merchant ships for Ptolemies and Seleucids

### PR DESCRIPTION
**Pull request by indoptogopt**
market, dock and merchant ships for Ptolemies and Seleucids
docks have texts about seafaring and maritime activity of that civ in general
markets have texts about overland trading
and merchant ships about maritime trading

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/market.xml [hunk 1]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>ptol</Civ>
     <SpecificName>Emporion</SpecificName>
+    <History>
+        The Ptolemaic dynasty of ancient Egypt was known for its extensive overland trading network, which connected the Mediterranean to the east and south. This trade network had been in place for centuries before the Ptolemies, but the Greek immigrants who settled in Egypt had a profound impact on the nature of trade in the southeast part of the known world. By securing strategic locations, such as cities and ports, along important travelling routes, either by forming alliances or by conquering, they maximized their access to goods  and recources that flowed through the Ptolemaic trading network.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/market.xml:11]:
```suggestion
        The Ptolemaic dynasty of Ancient Egypt was known for its extensive overland trading network, which connected the Mediterranean to  both the east and the south. This trade network had been in place for centuries before the Ptolemies, but the Greek immigrants who settled in Egypt had a profound impact on the nature of trade in the southeast part of the known world. By securing strategic locations, such as cities and ports, along important traveling routes, either by forming alliances or by conquering, they maximized their access to goods  and resources that flowed through the Ptolemaic trading network.
```

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/market.xml [hunk 2]
```diff
@@ -7,6 +7,10 @@
...
+        Ptolemaic merchants extensively traded in Africa, India and other parts of Asia, exporting agricultural products such as grain, wine and olive oil, textiles and spices in exchange for metals and exotic luxury items.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/market.xml:12]:
```suggestion
        Ptolemaic merchants extensively traded in Africa, India and other parts of Asia, exporting agricultural products such as grain, wine, olive oil, textiles, and spices in exchange for metals and exotic luxury items.
```

**Comment by TheShadowOfHassen** [comment]
can I merge this?

**Comment by indoptogopt** [comment]
Yes

**Pull request merged**